packages/ckeditor5-list/docs/features/lists.md

```diff
@@ -4,7 +4,7 @@ category: features
 
 # Lists
 
-The {@link module:list/list~List list} feature allows creating ordered (numbered) and unordered (bulleted) lists in the editor.
+The {@link module:list/list~List list} feature allows creating ordered (numbered) and unordered (bulleted) lists in the editor aimed at better structuring adn presenting specific content as well as serving special purposes, like tables of content or {@link features/todo-lists to-do lists}.
```

Comment:
typo in "adn"

Also I don't understand the second part of the sentence with tables. "Serving special purposes" means nothing.  Maybe you could give some examples what you need lists for? Organize content etc.?